packages/block-library/src/image/index.php

```diff
@@ -359,7 +359,7 @@ function register_block_core_image() {
 	);
 
 	if ( defined( 'IS_GUTENBERG_PLUGIN' ) && IS_GUTENBERG_PLUGIN ) {
```

Comment:
Should this conditional be removed once Modules API are in Core?

Reply:
Let's wait until [we've changed Webpack's Interactivity API](https://github.com/WordPress/wordpress-develop/blob/ba40e2809312337983b947fd365d037a281cd087/tools/webpack/blocks.js) to generate modules in Core.

Reply:
I've done this now that there is a PR for WP 6.5 where we can make the Webpack changes:

- https://github.com/WordPress/wordpress-develop/pull/5922

I'll make the necessary Webpack changes there.